manta-js/package/api/index.js

```diff
@@ -22,10 +18,10 @@ export class ApiConfig {
 
 export default class Api {
   // Constructs an API from a config
-  constructor(config) {
+  constructor(api,config) {
```

Comment:
come to think of it, we probably want to pass the suppressLogging flag into `Api` too